apps/stable/cluster-autoscaler/values.yaml.gotmpl

```diff
@@ -0,0 +1,28 @@
+{{- if eq "eks" .Values.jxRequirements.cluster.provider }}
+autoDiscovery: 
+  clusterName: {{ .Values.jxRequirements.cluster.provider }}
+  tags:
```

Comment:
@anilkumarpasupuleti we only need to add values that we override. Just the values I specified in the previous comments will do. Specifying values like `image.tag` is definitely unnecessary. That is doubling up on the chart version itself